index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Amr Ahmed - Portfolio</title>
+</head>
+<body>
+    <img src="/images/top_circles.png" alt="Background image circles white">
+    <!-- <img src="/images/portfoliobg.png" alt="Background image circles white"> -->
+    <div class="wrapper">
+        <div class="header">
+            <span>Amr Ahmed</span> <div class="hamburger-menu"><i class="fas fa-bars"></i></div>
```

Comment:
- [ ] Please ensure that your logo is wrapped in a link that directs you to the homepage. ie wrap is inside the `<a>` tag.
- [ ] Please ensure that you wrap your hamburger icon inside the `<nav>` tag. This is so as to improve on the semantics of your HTML.